docs/source/etcd-data-model.rst

```diff
@@ -40,8 +40,19 @@ endpoints
   (e.g. if it has multiple vNICs). See :ref:`endpoint-data` for more.
 
 security profiles
-  A security profile is an object that encapsulates a specific security policy
-  that can be applied to endpoints. See :ref:`security-profile-data` for more.
+  A security profile is an object that encapsulates a specific set of security
```

Comment:
Simplify: 'A security profile encapsulates a set of security rules for an endpoint.'